src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -172,21 +166,25 @@ Amplify.configure({
 });
 ```
 
-### Recap
-At this point, your client is now configured to point to your AppSync API endpoint and you updated your authentication details based on the auth types you are using.
+</Accordion>
+
+<br/>
 
-[comment]: # (ADD TO RECAP Tie-in to real-world use and application of the topic discussed OR reiterate the benefit.)
+**Recap:** At this point, your client is now configured to point to your AWS AppSync API endpoint and you updated your authentication details based on the auth types you are using.
```

Comment:
Is there anything more we want to add to the Recap to illustrate a tie-in to real-world use or application?

Reply:
Maybe we can say also "Now you're ready to call your GraphQL API." Something that alludes to that this is a pre-requisite to get things set up.